app/views/crowdblog/admin/posts/index.html.slim

```diff
@@ -8,12 +8,14 @@
       ul.nav.nav-pills
         li class="#{@state ? '' : 'active'}"
           = link_to 'All', crowdblog.admin_posts_path
+        li class="#{@state == 'drafted' ? 'active' : ''}"
+          = link_to 'Drafted', crowdblog.admin_posts_by_state_path('drafted')
         li class="#{@state == 'finished' ? 'active' : ''}"
           = link_to 'Finished', main_app.admin_finished_posts_path('finished')
+        li class="#{@state == 'reviewed' ? 'active' : ''}"
+          = link_to 'Reviewed', main_app.admin_finished_posts_path('reviewed')
```

Comment:
why do you have 2 different types of named routes? ```admin_posts_by_state_path(state)``` and ```admin_finished_posts_path``` shouldn't you need to use the first one? ```admin_posts_by_state_pateh('reviewed')``` or change it to say ```admin_darfted_posts_path``` ....

Please check the consistency of your named routes ^^